specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2020-05-01-preview/cpim.json

```diff
@@ -0,0 +1,366 @@
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Describe the result of a successful operation.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureActiveDirectory/guestUsages/{resourceName}": {
+      "put": {
+        "operationId": "GuestUsages_Create",
+        "tags": [
+          "GuestUsages"
+        ],
+        "x-ms-examples": {
+          "GuestUsages_Create": {
+            "$ref": "./examples/GuestUsagesCreate.json"
+          }
+        },
+        "summary": "Creates a Guest Usages resource",
+        "description": "Creates a Guest Usages resource for the Microsoft.AzureActiveDirectory resource provider",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "name": "Resource",
+            "in": "body",
+            "description": "Guest Usages resource to be created",
+            "schema": {
+              "$ref": "#/definitions/GuestUsagesResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and the resource was created successfully",
+            "schema": {
+              "$ref": "#/definitions/GuestUsagesResource"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "GuestUsages_Update",
+        "tags": [
+          "GuestUsages"
+        ],
+        "x-ms-examples": {
+          "GuestUsages_Update": {
+            "$ref": "./examples/GuestUsagesUpdate.json"
+          }
+        },
+        "summary": "Updates a Guest Usages resource",
+        "description": "Updates a Guest Usages resource for the Microsoft.AzureActiveDirectory resource provider",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "name": "Resource",
+            "in": "body",
+            "description": "Guest Usages Resource to be updated",
+            "schema": {
+              "$ref": "#/definitions/GuestUsagesResource"
```

Comment:
looks like every property present in the schema can be modified via a PATCH.

Reply:
can you confirm?

Reply:
No. Only the tags can be modified in the schema and nothing else.

Reply:
The request body should only contain tags in that case.

Reply:
Done